RecursiveSum.js

```diff
@@ -0,0 +1,5 @@
+const ArraySum = (numbers) => {
+  if (numbers.length === 0) return 0;
+  const [first, ...rest] = numbers;
+  return first + ArraySum(rest);
+};
```

Comment:
Good!,
Maybe consider renaming `rest` parameter to something like `remainingNumbers` to make more descriptive